scripts/data/TransientState.js

```diff
@@ -28,6 +28,17 @@ const transientState = {
   "view": ""
 }
 
+////////////MAKE DEEP COPY OF transientState with spread operator///////////////////////////
+let transientState = {...initialTransientState}
+
+
+////////////FUNCTION TO RESET transientState back to initial/////////////////////////////
+export const resetTransientState = () => {
+    transientState = {...initialTransientState}
```

Comment:
This isn't resetting the transientState.userId value. Maybe we could just hard code it to be transientState.userId = 0?